### DISCUSSION TITLE
it look like a bug when a object inside group drag quickly on v6(ts)

### DISCUSSION THREAD

**Discussion by wuxingping** [fabricjs/fabric.js#8641] [General]
it looks like a bug when a object inside group drag quickly when out of range of the size of the group on v6(ts).it seems that has some delays when operate quickly.
![1675325403519](https://user-images.githubusercontent.com/41353266/216267974-b76b28bb-d051-4cbe-8302-40706ab30ab5.jpg)
![1675325617708](https://user-images.githubusercontent.com/41353266/216268339-3208e558-a1a2-476b-b294-e07b1f6e34ab.jpg)

**Comment by ShaMan123** [comment]
This looks like caching. Turn `objectCaching` to false and see.
I will need to see a video to be sure.

**Reply by wuxingping** [reply]
yes,you are right.the key of this problem is caching.it works when i Turn objectCaching of group to false.thank you so much.
